SPARQLWrapper/main.py

```diff
@@ -21,23 +22,23 @@ class SPARQLWrapperFormatter(
     pass
 
 
-def check_file(v):
+def check_file(v: str) -> str:
     if os.path.isfile(v):
         return v
     elif v == "-":
         return "-"  # stdin
     else:
-        raise argparse.ArgumentError("file '%s' is not found" % v)
+        raise argparse.ArgumentTypeError("file '%s' is not found" % v)
```

Comment:
Even this, with it fixing a bug that could easily come up, should ideally be a separate PR.

Reply:
934e334